.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  dispatch-update:
+    name: Dispatch Update
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/repository-dispatch@v2
```

Comment:
How hard would it be to create our own version of this action?  In general, I think it's a little sketchy to depend upon a 3rd-party GitHub action for a relatively simple task like this when a PAT is involved.